src/main/java/com/revature/services/PersonRoleLogic.java

```diff
@@ -0,0 +1,8 @@
+package com.revature.services;
+
+import com.revature.domain.PersonRole;
+
+public interface PersonRoleLogic {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Annotate the "PersonRoleLogic" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1609)